### ISSUE TITLE
get osx version to work

### ISSUE THREAD

**Issue by demiurg** [Applied-GeoSolutions/gippy#5]
This involves fixing wrong boost symbols. And potentially fixing linking. Helpful links:

conda/conda-build#279
https://github.com/conda/conda-build/blob/e8587bb49d47b1255d6137b273b45b94b71fb4e7/conda_build/post.py#L151
http://qin.laya.com/tech_coding_help/dylib_linking.html
http://stackoverflow.com/questions/19123623/python-runtime-library-dirs-doesnt-work-on-mac

**Comment by earthflat** [comment]
Hi,

I also have these boost errors when installing gippy on OS X 10.10. Would anybody like to explain how to fix these errors? I am not so deep into these techniques...

Thanks in advance,
FE

**Comment by ircwaves** [comment]
Are  you installing from `pip` or from `git`?

**Comment by earthflat** [comment]
I've tried both with the same errors occurring.

**Comment by demiurg** [comment]
Can you post the errors, and how you are installing, exactly? Are you using homebrew?

**Comment by earthflat** [comment]
I've attached the quite long error outputs as text files. One is for using pip and the other using setup.py from github download.

pip:
To produce this error I try to install hippy via 'pip install gippy' in a condo environment. Usually I work with python 3.4 but also tried with python 2.7. Both produce the same error.

setup.py:
Similar situation as with pip. I've used 'python setup.py build' to produce the error.

Yes, I use homebrew. Also the boost library is installed via home-brew and on on the path.

[pip_install_gippy.txt](https://github.com/Applied-GeoSolutions/gippy/files/106931/pip_install_gippy.txt)
[git_setuppy.txt](https://github.com/Applied-GeoSolutions/gippy/files/106946/git_setuppy.txt)

**Comment by ircwaves** [comment]
I believe we have only tested/used python2.7 with this library.  I'll note that in the README.

**Comment by earthflat** [comment]
I have also tested with python 2.7 as stated above. The errors are the same for both versions.
So did someone mange to install on OS X with python 2.7?

**Comment by demiurg** [comment]
Yes, I've worked on getting it to work on OSX with success because that's what I work on. I'm unfortunately in the middle of some things right now, but the general instructions are basically, install dependencies by homebrew, including boost and gdal and I'm not sure if also python2.7. Then virtualenv should be created. Then the repo should be cloned pip install gippy needs to run, inside the venv. 

When I look at this i, I will try to get it installed on a fresh osx system and make exact install instructions.
